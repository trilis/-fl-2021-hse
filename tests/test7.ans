: "2" does not exist in alphabet, line :12
