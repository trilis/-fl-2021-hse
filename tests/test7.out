Exception: invalid `to` state in transition